control.lua

```diff
@@ -1,21 +1,102 @@
 -- Bounds for expected behaviour: 0.0 > x > -1.0
--- Above 0: becomes FastKey
+-- Above 0: becomes FastKey (0 is default modifier when creating player unless changed by mods)

```

Comment:
Addition should be on a separate line? Is this even true? `MODIFIER = -0.5` further down in the file